### ISSUE TITLE
Different VMAF results for builtin and path-specified models

### ISSUE THREAD

**Issue by adamjw24** [Netflix/vmaf#823]
I recently switched to VMAF 2.1.1 and noticed that the "vmafosexec" app is deprecated. I started using the "vmaf" app, and as I understand from the scarce command-line help, it comes with embedded models, which have analogue names to the models specified in the model/ directory.

This works quite well for smaller sequences, but for UHD signals I see big deviations between the results.

Please find attached a console log showing the problem:

```
user@computer:/mnt/d/Development/results/workspace$ ../../vmaf/libvmaf/build/tools/vmaf -r ../../sequences/JVET/CatRobot1_3840x2160p_60_10_709_420.yuv -d CatRobot.yuv -w 3840 -h 2160 -p 420 -b 10 -m version=vmaf_4k_v0.6.1 --threads=8
VMAF version 8ba4a4b8
300 frames ⠉⠩ 1.76 FPS
"CatRobot.yuv" ended before "../../sequences/JVET/CatRobot1_3840x2160p_60_10_709_420.yuv".

vmaf_4k_v0.6.1: 85.938284
user@computer:/mnt/d/Development/results/workspace$ ../../vmaf/libvmaf/build/tools/vmaf -r ../../sequences/JVET/CatRobot1_3840x2160p_60_10_709_420.yuv -d CatRobot.yuv -w 3840 -h 2160 -p 420 -b 10 -m path=../../vmaf/model/vmaf_4k_v0.6.1.json --threads=8
VMAF version 8ba4a4b8
300 frames ⠉⠩ 1.77 FPS
"CatRobot.yuv" ended before "../../sequences/JVET/CatRobot1_3840x2160p_60_10_709_420.yuv".

../../vmaf/model/vmaf_4k_v0.6.1.json: 92.506037
```

When I create log files, I see that the generated features take exactly the same values, so it seems only the VMAF predictor is somehow different.

Why do the embedded models differ from the path-specified? Is there a "correct" way to conduct a VMAF experiment? Which number should be used for comparison with literature?

**Comment by kylophone** [comment]
Hi,

Had a quick look at the code and this indeed looks to be a copy/paste bug for the built-in 4k model. I will prepare a fix for this today.

https://github.com/Netflix/vmaf/blob/master/libvmaf/src/model.c#L84-L86

Thanks,
Kyle

**Comment by kylophone** [comment]
Fixed by #827.
